Client/src/Common/Submitter/BaseClientSubmitter.cs

```diff
@@ -833,8 +833,7 @@ public ICollection<GetResultIdsResponse.Types.MapTaskResult> GetResultIds(IEnume
 
         var taskIdInError = resultStatus.IdsError.Any()
                               ? resultStatus.IdsError[0]
-                              : resultStatus.IdsResultError[0]
-                                            .TaskId;
+                              : resultStatus.IdsResultError[0].TaskId;
```

Comment:
I don't think this should change

Reply:
Resharper 2023 on my laptop, changed it back